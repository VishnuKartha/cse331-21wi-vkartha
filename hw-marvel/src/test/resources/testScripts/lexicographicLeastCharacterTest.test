# Tests that the algorithm favors the path with the lexicographical least child

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/alphabet1.tsv

LoadGraph alpha1 alphabet1.tsv

FindPath alpha1 A B

FindPath alpha1 B A