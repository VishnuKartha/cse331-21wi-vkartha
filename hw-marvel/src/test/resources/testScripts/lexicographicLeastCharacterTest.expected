# Tests that the algorithm favors the path with the lexicographical least child

# To run this test, the marvel dataset must exist at
# hw-marvel/src/main/resources/data/alphabet1.tsv

loaded graph alpha1

path from A to B:
A to C via connection1
C to B via connection2

path from B to A:
B to C via connection2
C to A via connection1
